dapp/src/hooks/useEnhancedStatistics.ts

```diff
@@ -0,0 +1,28 @@
+import useStatistics from "./useStatistics"
+
+const useEnhancedStatistics = () => {
```

Comment:
I understand your point and agree that `useStatistics` and `useEnhancedStatistics` can be confusing. However, the previous solution did not allow us to use the features from `tanstack`. Currently, we do not check the status of the TVL fetch This causes our progress bar to [look strange ](https://discord.com/channels/1079490158639976609/1313061786362056777/1313425787038466090)when the app starts.

<img width="734" alt="Screenshot 2024-12-11 at 09 25 15" src="https://github.com/user-attachments/assets/3d492530-5f72-4066-953f-e5d4fc57ad1c">

I will revert to the previous solution to avoid confusion. But we will have to back to this hook soon.

https://github.com/thesis/acre/pull/914/commits/0d5d3e8c6f578fd7fbc47974c2db78111853763b